addons/addon-base-raas/packages/base-raas-services/lib/indexes/indexes-service.js

```diff
@@ -76,9 +82,19 @@ class IndexesService extends Service {
     );
 
     // Validate input
-    const [validationService] = await this.service(['jsonSchemaValidationService']);
+    const [validationService, awsAccountsService] = await this.service([
+      'jsonSchemaValidationService',
+      'awsAccountsService',
+    ]);
     await validationService.ensureValid(rawData, createSchema);
 
+    // Make sure the AWS Account actually exists
+    try {
+      await awsAccountsService.mustFind(requestContext, { id: rawData.awsAccountId });
+    } catch (err) {
+      throw this.boom.badRequest(`Incorrect AWS Account ID provided`, true).cause(err);
```

Comment:
Since you are not using any string expression, you might want to use single quote instead.  Otherwise, you might get an eslint issue.